deployments/arbitrum-goerli/usdc/deploy.ts

```diff
@@ -0,0 +1,69 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet } from '../../../src/deploy';
+
+const HOUR = 60 * 60;
+const DAY = 24 * HOUR;
+
+const GOERLI_TIMELOCK = '0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const trace = deploymentManager.tracer()
+  const ethers = deploymentManager.hre.ethers;
+
+  // pull in existing assets
+  const USDC = await deploymentManager.existing('USDC', '0x72A9c57cD5E2Ff20450e409cF6A542f1E6c710fc', 'arbitrum-goerli');
+  const WETH = await deploymentManager.existing('WETH', '0xe39ab88f8a4777030a534146a9ca3b52bd5d43a3', 'arbitrum-goerli');
+
+  // Deploy ArbitrumBridgeReceiver
+  const bridgeReceiver = await deploymentManager.deploy(
+    'bridgeReceiver',
+    'bridges/arbitrum/ArbitrumBridgeReceiver.sol',
+    []
+  );
+
+  // Deploy Local Timelock
+  const localTimelock = await deploymentManager.deploy(
+    'timelock',
+    'vendor/Timelock.sol',
+    [
+      bridgeReceiver.address, // admin
+      1 * DAY,                // delay
+      14 * DAY,               // grace period
+      12 * HOUR,              // minimum delay
+      30 * DAY                // maxiumum delay
```

Comment:
Let's adjust these to more suitable values for testnet (like we have for Mumbai/Optimism Goerli).